lib/rubicure/girl.rb

```diff
@@ -55,6 +55,21 @@ def attack!
       current_attack_message
     end
 
+    def birthday?(date = Date.today)
```

Comment:
Rails/Date: Do not use `Date.today` without zone. Use `Time.zone.today` instead.